readme.md

```diff
@@ -477,11 +477,11 @@ Make array a setoid:
 ```js
 Array.prototype.equals = (arr) => {
```

Comment:
I don't have strong feelings either way. There is a note in the style guide
(contributing.md) that arrow functions should always have parenthesis

On Mon, Jul 25, 2016, 9:17 AM hemanth.hm <notifications@github.com> wrote:

> In readme.md
> <https://github.com/hemanth/functional-programming-jargon/pull/66#discussion_r72094944>
> :
>
> > @@ -477,11 +477,11 @@ Make array a setoid:
> >  ```js
> >  Array.prototype.equals = (arr) => {
>
> No parens in functor
> <https://github.com/hemanth/functional-programming-jargon#functor>
> though, now is time to discuss the merits and demerits of parens ;) ?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/hemanth/functional-programming-jargon/pull/66/files/929a321a738796f25a204fbdc7a44eaf4885291b#r72094944>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAB-4OM8gdrbOmHT1MzvMZtNKZBzsSpPks5qZOGYgaJpZM4JUC-j>
> .
>